### PR TITLE
[Metricbeat] gcp: add GKE overview dashboard

### PR DESCRIPTION
**Pull request by endorama**
## What does this PR do?

Provide an overview dashboard for GCP GKE metricset.

## Why is it important?

The newly added GCP GKE metricset has been merged without a default dashboard to show collected
metrics. This is an enhancement that allow users to immediately inspect metrics gathered through
this metricset.

## Checklist

- [x] My code follows the style guidelines of this project
- [ ] ~~I have commented my code, particularly in hard-to-understand areas~~
- [x] I have made corresponding changes to the documentation
- [ ] ~~I have made corresponding change to the default configuration files~~
- [ ] ~~I have added tests that prove my fix is effective or that my feature works~~
- [ ] I have added an entry in `CHANGELOG.next.asciidoc` or `CHANGELOG-developer.next.asciidoc`.

## Author's Checklist

- [ ]

## How to test this PR locally

1. Boot a development stack version 7.15.0 or from `master`
2. Configure `metricbeat` to collect `gcp.gke` metrics (is disabled by default in 7.15.0)
3. Setup dashboard with `metricbeat setup --dashboards`
4. Look at the dashboard

## Related issues

- Follow up on #26824 
- Requires https://github.com/elastic/beats/pull/27613

## Screenshots

Screenshot 2021-08-26 at 18-23-41  Metricbeat GCP  GKE Overview - Elastic

### CODE REVIEW

**Comment by kaiyan-sheng** [comment]
Screenshot in this [folder](https://github.com/elastic/beats/tree/master/metricbeat/docs/images) under Metricbeat is required and also reference the dashboard screenshot in GCP [documentation](https://github.com/elastic/beats/blob/master/x-pack/metricbeat/module/gcp/_meta/docs.asciidoc) please 😄

**Comment by endorama** [comment]
I saw that there is ` module.yml` file in some modules where the dashboard is referenced. Should I add it?

**Review by MichaelKatsoulis — APPROVED** [review]
LGTM!

File: x-pack/metricbeat/module/gcp/_meta/kibana/7/dashboard/Metricbeat-gcp-gke-overview.ndjson [hunk 1]
```diff
@@ -0,0 +1,3 @@
...lyzed\":false,\"count\":0,\"doc_values\":true,\"indexed\":true,\"name\":\"ceph.cluster_status.misplace.total\",\"scripted\":false,\"searchable\":true,\"type\":\"number\"},{\"aggregatable\":true,\"analyzed\":false,\"count\":0,\"doc_values\":true,\"indexed\":true,\"name\":\"ceph.cluster_status.misplace.objects\",\"scripted\":false,\"searchable\":true,\"type\":\"number\"},{\"aggregatable\":true,\"analyzed\":false,\"count\":0,\"doc_values\":true,\"indexed\":true,\"name\":\"ceph.cluster_status.misplace.ratio\",\"scripted\":false,\"searchable\":true,\"type\":\"number\"},{\"aggregatable\":true,\"analyzed\":false,\"count\":0,\"doc_values\":true,\"indexed\":true,\"name\":\"ceph.cluster_status.degraded.total\",\"scripted\":false,\"searchable\":true,\"type\":\"number\"},{\"aggregatable\":true,\"analyzed\":false,\"count\":0,\"doc_values\":true,\"indexed\":true,\"name\":\"ceph.cluster_status.degraded.objects\",\"scripted\":false,\"searchable\":true,\"type\":\"number\"},{\"aggregatable\":true,\"analyzed\":false,\"count\":0,\"doc_values\":true,\"indexed\":true,\"name\":\"ceph.cluster_status.degraded.ratio\",\"scripted\":false,\"searchable\":true,\"type\":\"number\"},{\"aggregatable\":true,\"analy
```
Comment (kvch) [inline x-pack/metricbeat/module/gcp/_meta/kibana/7/dashboard/Metricbeat-gcp-gke-overview.ndjson:3]: This line is not supposed to be here. If you set `excludeExportDetails` to `false` these lines are not added to the Kibana response. Did you use the export command of Beats?

Comment (endorama) [reply x-pack/metricbeat/module/gcp/_meta/kibana/7/dashboard/Metricbeat-gcp-gke-overview.ndjson:3]: No, as is not working at the moment for me, so I exported them via the Kibana Saved Object UI. So I (wrongly) assumed that the format was the same?

This is how I used the export command:
- pulled latest master
- `mage update`, `mage build` in `x-pack/metricbeat`
- "./metricbeat export dashboard -id uuid >> file.ndjson`

**Comment by endorama** [comment]
https://github.com/elastic/beats/pull/27613 was merged, conflicts resolved, CI happy. Merging

**Pull request merged**